modules/magniteAnalyticsAdapter.js

```diff
@@ -633,6 +658,47 @@ const handleBidWon = args => {
   addEventToQueue({ bidsWon: [bidWon] }, bidWon.renderAuctionId, 'bidWon');
 }
 
+window.addEventListener('dmWebVitals', eventObj => {
+  let dmWebVitals = eventObj.detail;
+  dmWebVitals.accountId = accountId;
+  addEventToQueue({ dmWebVitals }, dmWebVitals.auctionId, 'webVital');
+});
+
+const countPendingEvents = (missedEvents, pendingEvents) => {
```

Comment:
semantic note: this function is not _counting_... can we change the name to `addPendingEventsToMissedEvents`? or something else?

Reply:
yes I am bad at naming things